WooCommerce/src/main/kotlin/com/woocommerce/android/model/Product.kt

```diff
@@ -98,7 +98,9 @@ data class Product(
         val name: String,
         val options: List<String>,
         val isVisible: Boolean
-    ) : Parcelable
+    ) : Parcelable {
+        fun isLocalAttribute() = id == 0L
```

Comment:
```suggestion
        val isLocalAttribute: Boolean
            get() = id == 0L
```
NP: It's more idiomatic in Kotlin to use a property for simple values like this.